src/Elastic.Apm.SqlClient/SqlEventListener.cs

```diff
@@ -0,0 +1,149 @@
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Diagnostics.Tracing;
+using Elastic.Apm.Api;
+using Elastic.Apm.Logging;
+using Elastic.Apm.Model;
+
+namespace Elastic.Apm.SqlClient
+{
+	internal class SqlEventListener : EventListener
+	{
+		private readonly ApmAgent _apmAgent;
+		private readonly IApmLogger _logger;
+
+		private readonly ConcurrentDictionary<int, (Span Span, long Start)> _processingSpans = new ConcurrentDictionary<int, (Span, long)>();
+
+		public SqlEventListener(IApmAgent apmAgent)
+		{
+			_apmAgent = (ApmAgent)apmAgent;
+			_logger = _apmAgent.Logger.Scoped(nameof(SqlEventListener));
+		}
+
+		private EventSource _eventSource;
```

Comment:
nit: I think this is a leftover - could you please remove?

Reply:
fixed